LeaderboardBackend/Properties/launchSettings.json

```diff
@@ -1,31 +1,40 @@
-﻿{
-	"$schema": "https://json.schemastore.org/launchsettings.json",
-	"iisSettings": {
-		"windowsAuthentication": false,
-		"anonymousAuthentication": true,
-		"iisExpress": {
-			"applicationUrl": "http://localhost:48031",
-			"sslPort": 44351
-		}
-	},
-	"profiles": {
-		"LeaderboardBackendCsPoc": {
-			"commandName": "Project",
-			"dotnetRunMessages": true,
-			"launchBrowser": true,
-			"launchUrl": "swagger",
-			"applicationUrl": "https://localhost:7128;http://localhost:5240",
-			"environmentVariables": {
-				"ASPNETCORE_ENVIRONMENT": "Development"
-			}
-		},
-		"IIS Express": {
-			"commandName": "IISExpress",
-			"launchBrowser": true,
-			"launchUrl": "swagger",
-			"environmentVariables": {
-				"ASPNETCORE_ENVIRONMENT": "Development"
-			}
-		}
-	}
+{
+  "profiles": {
+    "LeaderboardBackendCsPoc": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "dotnetRunMessages": true,
+      "applicationUrl": "https://localhost:7128;http://localhost:5240"
+    },
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "Docker (Production)": {
+      "commandName": "Docker",
+      "publishAllPorts": true,
+      "httpPort": 8000,
+      "useSSL": false,
```

Comment:
Is this only temporary, or is the plan to rely on some other configuration to use SSL?

Reply:
SSL does not work with the container because it needs a certificate file. It's annoying to setup locally so I skipped it.

And anyway in the dev env https is handled automatically handled by the platform's load balancer. I expect that we will handle https in the same way in production.